Sources/PackagePlugin/PublicAPI/Command.swift

```diff
@@ -41,15 +58,15 @@ public final class CommandConstructor {
     ///     input files cause the command to be rerun.
     ///   - outputFiles: Output files that should be processed further according
     ///     to the rules defined by the build system.
-    public func addBuildCommand(
+    static func buildCommand(
```

Comment:
do these need to be public?

Reply:
Yes, they do.  Good catch.  I'll extend the unit test to use them.

Reply:
Actually they don't because the whole protocol is public.  And this is being tested.